go.mod

```diff
@@ -86,6 +86,98 @@ require (
 	gopkg.in/guregu/null.v4 v4.0.0
 )
 
+require (
```

Comment:
why do we require this change?

Reply:
Sorry...I ran go get at the root level instead of changing directory first.